### PR TITLE
Switching between sites + MODIFY ASSET BUG

### PR DESCRIPTION
**Pull request by juliasaveliff**

### CODE REVIEW

File: rackcity/migrations/0052_merge_20200416_1547.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Generated by Django 3.0.3 on 2020-04-16 19:47
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
```
Comment (juliasaveliff) [inline rackcity/migrations/0052_merge_20200416_1547.py:6]: nobody ever migrated this poor bby

File: rackcity/models/asset.py [hunk 2]
```diff
@@ -137,10 +137,6 @@ def validate_location_type(
                 "Rackmount assets must not have a chassis or chassis slot. "
             )
     else:
-        if offline_storage_site and (chassis or chassis_slot):
```
Comment (juliasaveliff) [inline rackcity/models/asset.py:140]: per discussions with Benjamin, blades in offline storage will stay in their chassis

File: rackcity/utils/rackcity_utils.py [hunk 3]
```diff
@@ -58,6 +58,8 @@ def validate_asset_location_in_rack(
     change_plan=None,
     related_asset_id=None,
 ):
+    if asset_rack_position is None:
+        return
     new_asset_location_range = [asset_rack_position + i for i in range(asset_height)]
```
Comment (juliasaveliff) [inline rackcity/utils/rackcity_utils.py:63]: THIS LIST COMPREHENSION WAS THROWING AN EXCEPTION DEEP IN THE DEPTHS OF ASSET SAVING

File: rackcity/models/asset.py (hunk 2 above)
Comment (benauriemma) [reply rackcity/models/asset.py:140]: I think they can be either in or out of their chassis?

Comment (juliasaveliff) [reply rackcity/models/asset.py:140]: yeah that's wha ti meant

**Pull request merged**